web/ui/package.json

```diff
@@ -5,6 +5,9 @@
   "repository": "https://github.com/control-center/serviced",
   "private": true,
   "devDependencies": {
+    "angular-auth0": "^3.0.0",
+    "auth0": "^2.9.1",
+    "auth0-js": "^9.4.2",
```

Comment:
If we were adding these deps to `package.json`, they would not belong in `devDependencies` because they are actually runtime deps, not build time, so they should be removed.